dynawo/sources/Modeler/DataInterface/IIDM/DYNGeneratorInterfaceIIDM.cpp

```diff
@@ -277,6 +277,16 @@ GeneratorInterfaceIIDM::getActivePowerControlDroop() const {
   return 0.;
 }
 
+bool
+GeneratorInterfaceIIDM::hasCoordinatedReactiveControl() const {
+  return false;
```

Comment:
Why always false ?

Reply:
because this extension does not exist for older iidm versions